### PR TITLE
Refs [TASK] [37433] Khởi tạo project

### PR DESCRIPTION
**Pull request by luongth-2520**
## Related Tickets
- [#Task 37433: Khởi tạo project](https://edu-redmine.sun-asterisk.vn/issues/37433)
## WHAT
- Khởi tạo project
## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://github.com/framgia/coding-standards/tree/master/eng/android |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Redmine | Sun* Redmine working process  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md |<li>- [ ] yes</li>|<li>- [ ] yes</li>

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        android:theme="@style/Theme.Unsplash03">
+        <activity android:name=".MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+    </application>
+
```
Comment (ghost) [inline app/src/main/AndroidManifest.xml]: remove space

Comment (khoaqh-2470) [inline app/src/main/AndroidManifest.xml]: remove line

**Review by khoaqh-2470 — APPROVED** [review]

File: app/src/main/java/com/sun/unsplash03/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package com.sun.unsplash03
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/unsplash03/MainActivity.kt]:
```suggestion
class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
```

File: app/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="purple_200">#FFBB86FC</color>
+    <color name="purple_500">#FF6200EE</color>
+    <color name="purple_700">#FF3700B3</color>
+    <color name="teal_200">#FF03DAC5</color>
+    <color name="teal_700">#FF018786</color>
+    <color name="black">#FF000000</color>
+    <color name="white">#FFFFFFFF</color>
```
Comment (tuanlvt-1117) [inline app/src/main/res/values/colors.xml]: 2 màu này đã có sẵn trong system

**Review by ghost — APPROVED** [review]

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**